types/node/index.d.ts

```diff
@@ -647,6 +647,14 @@ declare namespace NodeJS {
         system: number;
     }
 
+    export interface ProcessRelease {
+        name: string;
+        sourceUrl: string;
+        headersUrl: string;
+        libUrl: string;
```

Comment:
This should be `libUrl?: string` also generally use the `?` over `undefined`.

Reply:
> libUrl is only present on Windows builds of Node.js and will be missing on all other platforms.

Reply:
In custom builds from non-release versions of the source tree, only the > name property may be present. The additional properties should not be relied upon to exist.